src/main/webapp/script.js

```diff
@@ -431,7 +434,8 @@ function getOutdatedSDK(aggregated) {
   var container = document.getElementById('sdkAnalysis');
   container.innerHTML += '<h3>The following jobs are using outdated SDKs.</h3>';
   for (outdatedJob in aggregated['STALE']) {
-    container.innerHTML += '<p>'+JSON.stringify(aggregated['STALE'][outdatedJob].name)+'</p>';
+    container.innerHTML += '<p>'+JSON.stringify(aggregated['STALE'][outdatedJob].name).replace(/\"/g, "")+'</p>';
+    container.innerHTML += '<p class="sdkDetails">'+JSON.stringify(aggregated['STALE'][outdatedJob].sdkName).replace(/\"/g, "")+' ('+JSON.stringify(aggregated['STALE'][outdatedJob].sdk).replace(/\"/g, "")+')<p>';
```

Comment:
You should add spaces before and after +.
`container.innerHTML += '<p>' + JSON.stringify(aggregated['STALE'][outdatedJob].name) + '</p>';
 container.innerHTML += '<p>' + JSON.stringify(aggregated['STALE'][outdatedJob].name).replace(/\"/g, "") + '</p>';
 container.innerHTML += '<p class="sdkDetails">' + JSON.stringify(aggregated['STALE'][outdatedJob].sdkName).replace(/\"/g, "") + ' (' + JSON.stringify(aggregated['STALE'][outdatedJob].sdk).replace(/\"/g, "") + ')<p>';`

Reply:
Good point! Changed in 7c2aea6.